util/check_tool_requirements.py

```diff
@@ -308,7 +328,8 @@ def dict_to_tool_req(path, tool, raw):
         'edalize': PyModuleToolReq,
         'vcs': VcsToolReq,
         'verible': VeribleToolReq,
-        'verilator': VerilatorToolReq
+        'verilator': VerilatorToolReq,
+        'vivado': VivadoToolReq
```

Comment:
Please add a trailing comma here to make the next diff easier.

Reply:
Yep, will do.